shaders/src/depth_main.fs

```diff
@@ -47,6 +47,7 @@ void main() {
     fragColor.xy = computeDepthMomentsVSM(depth);
     fragColor.zw = computeDepthMomentsVSM(-1.0 / depth); // requires at least RGBA16F
 #elif defined(VARIANT_HAS_PICKING)
+    initObjectUniforms(object_uniforms); // only needed for picking
```

Comment:
I would be more confortable moving this just under the `filament_lodBias - ` line.

I think it's okay to protect it with `#ifdef VARIANT_HAS_PICKING` but with a comment stating that currently it's only used in that case. But I think it'll be safer to not even do that and let the compiler eliminate the unneeded code.